Makefile

```diff
@@ -26,6 +26,7 @@ pull-monitoring:
 	go build -o pull-monitoring cmd/monitoring.go
 
 output/dashboards: pull-monitoring
+# set NOPULL=1 to disable pulling configs
 	bash scripts/prepare_dashboards.sh
```

Comment:
consider use a standalone task: it is no need to build the tool when no need to pulling the configs.